packages/content-conformance/src/config.ts

```diff
@@ -83,3 +98,32 @@ function applyConfigDefaults(config: ContentConformanceConfig) {
 function resolveConfigRoot(rootValue: string, cwd: string) {
   return path.resolve(cwd, rootValue)
 }
+
+/**
+ * Attempts to load a preset, if `config.preset` is defined, and merge them. The loaded config takes precedence over the preset.
+ */
+async function resolveConfigPreset(
+  config: ContentConformanceConfig,
+  { cwd }: { cwd: string }
+) {
+  if ('preset' in config) {
+    const loadedPreset = await loadConfig({
+      cwd,
+      pathToConfigOrPresetName: config.preset,
+      shouldValidateConfig: false,
+    })
```

Comment:
Do you think we should `catch` this and explicitly log something like 👇  before throwing?

> [content-conformance] No such preset

?

...Otherwise, it looks like we'd get a generic error from `loadConfig`, which might be confusing to resolve
> [content-conformance] Config file not found

Reply:
Addressed in [`edacc74` (#127)](https://github.com/hashicorp/web-platform-packages/pull/127/commits/edacc74f4dcdef159e4dad56fdb7e81685c60963)